app/uk/gov/hmrc/transitmovementsvalidator/controllers/MessagesController.scala

```diff
@@ -53,8 +54,11 @@ class MessagesController @Inject() (cc: ControllerComponents, validationService:
           validationService
             .validateXML(messageType, request.body)
             .map {
-              case Left(x)  => BadRequest(Json.toJson(BaseError.badRequestError(x)))
-              case Right(x) => Ok(Json.toJson(ValidationResponse(x)))
+              case Left(x)  => x.head match {
+                case UnknownMessageTypeValidationError(m) => BadRequest(Json.toJson(BaseError.badRequestError(m)))
+                case _ => Ok(Json.toJson(ValidationResponse(x)))
```

Comment:
I'll preface this with "I know it's a draft" - but thinking on this - I know you were considering sending an empty body if validation succeeded. I was a little bit rettisent about that because it'd make my parsing hard, but I realised that there is a status code for that - 204 No Content. Perhaps that's the way forward? Though it feels a bit weird sending a 204 for everything was fine, and a 200 Ok for we valiated but something wasn't fine - though it makes sense in the RESTful world, I guess.

Reply:
@crystalmark what are you thoughts?

Reply:
Resolved by adding a 204